dashboard/test/lib/services/registration_reminder_test.rb

```diff
@@ -18,10 +18,9 @@ class Services::RegistrationReminderTest < ActiveSupport::TestCase
       Services::RegistrationReminder.send_registration_reminders!
       assert_empty application.emails.where(email_type: 'registration_reminder')
 
-      # Fake sending first accepted email by updating the status to accepted and setting sent_at field
+      # Fake send first accepted email
       Timecop.travel 1.day
-      application.update!(status: 'accepted')
```

Comment:
Since the email gets sent from the controller now instead of from the model, updating the status directly no longer triggers an email.